patches/chrome-BUILD.gn.patch

```diff
@@ -66,6 +66,24 @@ index b8c378b044d9142c9936c97e686fb6bfcb37c918..38e96be506e2ba20aa421c8a7b0e051f
            "-v",
            rebase_path(chrome_version_file, root_build_dir),
            "-g",
+@@ -781,7 +794,7 @@ if (is_win) {
+         # framework itself, that would cause a cyclical dependency. Instead,
+         # this dependency directly copies the file into the framework's
+         # resources directory.
+-        public_deps += [ ":chrome_framework_widevine_signature" ]
```

Comment:
can you explain these? I'm not sure if there's a better option yet, but I'd like to look at it more closely

Reply:
@bridiver This target is for generating signature file from unsigned framework dylib for widevine cdm host verification.
We shouldn't generate sig file here because signature file should be generated from signed dylib.
So, our generation is done in at https://github.com/brave/brave-core/blob/master/patches/chrome-installer-mac-signing-signing.py.patch#L44 after signing framework bundle.